### ISSUE TITLE
FR - Resources - Track Templates - Save selected template from selected track

### ISSUE THREAD

**Issue by Jeff0S** [reaper-oss/sws#473]
_From [greg.so...@gmail.com](https://code.google.com/u/115091438113147976777/) on May 28, 2012 08:16:41_

hey guys, here's a thing I run into often that I thought you might be interested in implementing.

I use track templates a lot, and often I'll save one, and the down the road I want to save it again. So what I do is delete the original, save the new one, and rename it. Though what I thought might be nice is if a selected slot (track templates) coule be right clicked, and in the context menu is an option

"save slot\template from currently selected track"

Basically writing over that track template with the track selected in the project. This would be quite a bit smoother than - delete, save, rename - 

Maybe it's something worth thinking over, as it might have applications for the other resource types as well - 

cheers!

greg

_Original issue: http://code.google.com/p/sws-extension/issues/detail?id=473_
